airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml

```diff
@@ -41,10 +41,10 @@ acceptance_tests:
   discovery:
     tests:
       - backward_compatibility_tests_config:
-          disable_for_version: 0.2.3
+          disable_for_version: 0.2.5
         config_path: secrets/config.json
       - backward_compatibility_tests_config:
-          disable_for_version: 0.2.3
+          disable_for_version: 0.2.5
```

Comment:
Why do we need to disable the backwards compatibility tests if its not a breaking change?

Reply:
Correct me if, I'm wrong, but changing the `format` in the schema leads to a crash of CAT but this is not a braking change for the Customer. No?

Reply:
@erohmensing It's actually not needed. I've removed the `backward_compatibility_tests_config` from `acceptance-test-config.yaml`, the changes are not breaking.

Reply:
Thanks, makes me feel better!